codecov_auth/views/gitlab.py

```diff
@@ -40,11 +41,16 @@ def get_url_to_redirect_to(self):
         redirect_info = self.redirect_info
         base_url = urljoin(redirect_info["repo_service"].service_url, "oauth/authorize")
         state = self.generate_state()
+
+        scope = get_config("gitlab_scopes", default="api")
```

Comment:
Let's add this config it to settings base, and use it in redirect info, it's easier to tweak/follow convention here